services/web/server/src/simcore_service_webserver/projects/models.py

```diff
@@ -22,6 +29,43 @@ def to_project_type_db(cls, api_type: "ProjectTypeAPI") -> ProjectType | None:
         }[api_type]
 
 
+class ProjectDB(BaseModel):
+    id: str
+    type: ProjectType
+    uuid: ProjectID
+    name: str
+    description: str | None
+    thumbnail: HttpUrlWithCustomMinLength | None
+    prj_owner: int
+    creation_date: datetime
+    last_change_date: datetime
+    access_rights: dict[GroupIDStr, AccessRights]
+    ui: StudyUI | None
+    classifiers: list[ClassifierID]
+    dev: dict | None
+    quality: dict[str, Any]
+    published: bool
+    hidden: bool
+
+    class Config:
+        orm_mode = True
+
+
+assert set(ProjectDB.__fields__.keys()).issubset(  # nosec
+    {c.name for c in projects.columns}
+)
+
+
+class UserProjectAccessRightsDB(BaseModel):
+    uid: UserID
```

Comment:
this is supposed to represent the project access rights? this should be a GroupID not a UserID. At least that is what I see in one glance in the project table. let's check together.